docs/install/kubernetes.md

```diff
@@ -3,149 +3,143 @@ title: "Kubernetes"
-    kubectl get storageclasses | grep "storageos"
-    ```
+```bash
+helm repo add ondat https://ondat.github.io/charts && \
+helm repo update && \
+```
 
-### Option B - Using An External `etcd` Deployment
+2. The last command installs Ondat with a set of basic install parameters that are sufficient for a basic trial installation.
 
-#### Step 1 - Setup An `etcd` Cluster
+```bash
+helm install ondat ondat/ondat \
+  --namespace=storageos \
+  --create-namespace \
+  --set ondat-operator.cluster.portalManager.enabled=true \
+  --set ondat-operator.cluster.portalManager.clientId=37540b25-285c-4326-b76c-742100723ac3 \
+  --set ondat-operator.cluster.portalManager.secret=e946f84a-e6c0-4afd-9087-f9cdd6906aa5 \
+  --set ondat-operator.cluster.portalManager.apiUrl=https://portal-setup-7dy4neexbq-ew.a.run.app \
+  --set ondat-operator.cluster.portalManager.tenantId=16e51eb9-37cf-4103-9a31-9e2cdaeec373 \
+  --set etcd-cluster-operator.cluster.replicas=3 \
+  --set etcd-cluster-operator.cluster.storage=6Gi \
+  --set etcd-cluster-operator.cluster.resources.requests.cpu=100m \
+  --set etcd-cluster-operator.cluster.resources.requests.memory=300Mi
+```
 
-* Ensure that you have an `etcd` cluster deployed first before installing Ondat. For instructions on how to set up an external `etcd` cluster, review the [`etcd` documentation](https://docs.ondat.io/docs/prerequisites/etcd/#production---etcd-on-external-virtual-machines) page.
-* Once you have an `etcd` cluster up and running, ensure that you note down the list of `etcd` endpoints as comma-separated values that will be used when configuring Ondat in **Step 3**.
-  * For example, `203.0.113.10:2379,203.0.113.11:2379,203.0.113.12:2379`
+3. The installation process may take a few minutes. The end of this guide contains information on verifying the installation and licensing.
 
-#### Step 2 - Conducting Preflight Checks
+### Step 3b - Installing via kubectl-storageos plugin
 
-* Run the following command to conduct preflight checks against the Kubernetes cluster to validate that Ondat prerequisites have been met before attempting an installation.
+The Ondat Portal UI will display the following cmd that can be used to install Ondat using the `kubectl-storageos` plugin
 
-    ```bash
-    kubectl storageos preflight
-    ```
+![kubectl-storageos Install](/images/docs/install/PluginInstall.png)
 
-#### Step 3 - Installing Ondat
+This command uses the `kubectl-storageos` plugin command with a set of basic install parameters that are sufficient for a basic trial installation. The installation process may take a few minutes.
 
-1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance. In addition, define and export a `ETCD_ENDPOINTS` environment variable, where the value will be a list of `etcd` endpoints as comma-separated values noted down earlier in **Step 2**.
+```bash
+kubectl storageos install \
+  --include-etcd=true \
```

Comment:
We are missing the etcd storage class, or a mention that a default StorageClass needs to exist.

Reply:
The installation of the StorageClass is added to Prerequisites Section 3 - Installing a Local Path Provisioner